### PR TITLE
app-benchmarks/geekbench: New package

### PR DESCRIPTION
**Pull request by ConiKost**
app-benchmarks/geekbench: New package

Closes: https://bugs.gentoo.org/524652

### CODE REVIEW

**Comment by monsieurp** [comment]
Thank you @ConiKost for the PR.

Hi @ulm! Could you take a look at this project's license and tell us whether it fits into Portage? Thanks in advance.

**Comment by ulm** [comment]
- This will need bindist restriction too (in addition to fetch restriction).
- The license should be added to the EULA group in profiles/license_groups.

**Comment by ConiKost** [comment]
@ulm: Done, thanks!

**Comment by ConiKost** [comment]
@mgorny: Would you have a look? Thanks!

File: app-benchmarks/geekbench/geekbench-2.4.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+DESCRIPTION="A Cross-Platform Benchmark for Android, iOS, Linux, MacOS and Windows"
+HOMEPAGE="https://www.geekbench.com"
+SRC_URI="https://cdn.primatelabs.com/Geekbench-${PV}-Linux.tar.gz"
+
+KEYWORDS="-* ~amd64 ~x86"
+LICENSE="geekbench"
+SLOT="2"
+
+RDEPEND="sys-devel/gcc
```
Comment (mgorny) [inline app-benchmarks/geekbench/geekbench-2.4.3.ebuild]: Why no RDEP in 2.4.2?

Comment (ConiKost) [reply app-benchmarks/geekbench/geekbench-2.4.3.ebuild]: ldd says, 2.4.2 for arm is static. 2.4.3, 3.4.2 and 4.2.2 are not static.

Comment (mgorny) [reply app-benchmarks/geekbench/geekbench-2.4.3.ebuild]: Well, that's good enough argument for me. From my experience static executables rarely work cross-libc (contrary to what people using them think) but not our problem here.

**Comment by mgorny** [comment]
```
 * QA Notice: Files built without respecting CFLAGS have been detected
 *  Please include the following list of files in your report:
 * /opt/geekbench2/geekbench_arm_32
 * /opt/geekbench2/geekbench
```

Please look `man 5 ebuild` up for `QA_PREBUILT`. You need to use that to skip checks for prebuilt executables.

File: app-benchmarks/geekbench/geekbench-2.4.3.ebuild (hunk 1 above)
Comment (mgorny) [reply app-benchmarks/geekbench/geekbench-2.4.3.ebuild]: Actually, `readelf -d` lists the libraries for me (in *_arm_32).

Comment (ConiKost) [reply app-benchmarks/geekbench/geekbench-2.4.3.ebuild]: I see, checked myself, I've added RDEPEND.

**Comment by ConiKost** [comment]
@mgorny: I've added QA_PREBUILT now for v2 and v3.

**Comment by mgorny** [comment]
Thanks!

**Pull request closed without merging**